admin/attributes_controller.php

```diff
@@ -1572,15 +1581,15 @@ function popupWindow(url) {
                           <i class="fa-solid fa-circle fa-stack-1x txt-orange" aria-hidden="true"></i>
                           <i class="fa-regular fa-circle fa-stack-1x" aria-hidden="true"></i>
                         </span>
-                      <?php } ?>
-                      <?php echo $values_name; ?>
+                      <?php }
+                      echo $values_name ?>
                     </td>
-                    <td class="text-right"><?php echo $attributes_value['price_prefix']; ?>&nbsp;<?php echo $attributes_value['options_values_price'] . $attribute_has_wholesale; ?></td>
-                    <td class="text-right"><?php echo $attributes_value['products_attributes_weight_prefix']; ?>&nbsp;<?php echo $attributes_value['products_attributes_weight']; ?></td>
-                    <td class="text-right"><?php echo $attributes_value['products_options_sort_order']; ?></td>
+                    <td class="text-right"><?= $attributes_value['price_prefix']; ?>&nbsp;<?= $attributes_value['options_values_price'] . $attribute_has_wholesale ?></td>
+                    <td class="text-right"><?= $attributes_value['products_attributes_weight_prefix']; ?>&nbsp;<?= $attributes_value['products_attributes_weight'] ?></td>
+                    <td class="text-right"><?= $attributes_value['products_options_sort_order'] ?></td>
                     <?php if ($action == '') { ?>
                       <td class="text-center">
-                        <a href="<?php echo zen_href_link(FILENAME_ATTRIBUTES_CONTROLLER, 'action=set_flag_attributes_display_only' . '&attributes_id=' . $attributes_value['products_attributes_id'] . '&' . ($currentPage != 0 ? 'page=' . $currentPage . '&' : '') . 'products_filter=' . $products_filter . '&current_category_id=' . $current_category_id . '&flag=' . $attributes_value['attributes_display_only']); ?>" onClick="divertClick(this.href);return false;" title="<?php echo LEGEND_ATTRIBUTES_DISPLAY_ONLY; ?>">
+                        <a href="<?= zen_href_link(FILENAME_ATTRIBUTES_CONTROLLER, 'action=set_flag_attributes_display_only' . '&attributes_id=' . $attributes_value['products_attributes_id'] . '&' . ($currentPage != 0 ? 'page=' . $currentPage . '&' : '') . 'products_filter=' . $products_filter . '&current_category_id=' . $current_category_id . '&flag=' . $attributes_value['attributes_display_only']); ?>" onClick="divertClick(this.href);return false;" title="<?= LEGEND_ATTRIBUTES_DISPLAY_ONLY ?>">
                           <span class="fa-stack">
```

Comment:
Remove trailing semi-colon (`$attributes_value['attributes_display_only']);`).

Also, for different variables, on lines 1603, 1614, 1625, 1636 and 1647.